rest-model/src/main/java/com/gentics/mesh/core/rest/node/NodeUpdateRequest.java

```diff
@@ -20,9 +20,9 @@
 	@JsonPropertyDescription("Dynamic map with fields of the node content.")
 	private FieldMap fields = new FieldMapImpl();
 
-	@JsonProperty(required = true)
-	@JsonPropertyDescription("Version number which must be provided in order to handle and detect concurrent changes to the node content.")
-	private String version;
+	@JsonProperty(required = false)
+	@JsonPropertyDescription("Version number which can be provided in order to handle and detect concurrent changes to the node content.")
```

Comment:
Don't handle the default in the POJO. Instead update the value within the REST handler.